### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by CheeseTastisch**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,29 +1,4 @@
...
-
-## How to contribute?
-If you want to contribute to QuillORM, you can do so by creating a pull request.
-
-However, in the current phase of development, where a concept is developed, we try to keep the team as small as possible,
-to keep the basic view of QuillORM in sight and make development as fast as possible.
-
-## Concept
-You can view the current concept at [Concept.md](https://github.com/CheeseTastisch/QuillORM/blob/main/concept/Concept.md).
+This repository does not contain any other content.
```
Comment (SIMULATAN) [inline README.md:4]: If my comment on #3 regarding the concept is accepted, said concept will probably live in this repository.

Comment (CheeseTastisch) [reply README.md:4]: Let's keep this PR open for now and wait for the outcome on #3.

Comment (CheeseTastisch) [reply README.md:4]: As #3 is solved now, i think we can keep that for now.

**Review by SIMULATAN — APPROVED** [review]
approved
yes

**Pull request merged**